internal/handlers/handlers.go

```diff
@@ -0,0 +1,104 @@
+// Package handlers contains functions for converting generic GCP PubSub messages to app-specific messages
+package handlers
+
+import (
```

Comment:
Nit: group imports (stdlib, 3rd party, same project)?

Ditto other files

Reply:
As of right now I let IntelliJ sort the imports. I'll add a note to configure a linter and precommit as I'm not going to manually sort the imports or add a linter+precommit as a part of this change.